captainhook.json

```diff
@@ -0,0 +1,57 @@
+{
+    "commit-msg": {
+        "enabled": true,
```

Comment:
This one needs to be disabled too from the start. It needs to be enabled only when the project enters into support phase.